.changeset/twenty-rocks-end.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-badge': major
+---
```

Comment:
@alexcarpenter - I was thinking about updating the changeset to a `minor` update. Looking at the old component, it doesn't introduce any breaking changes. Does that sound reasonable?

Reply:
Works for me. Should be an easy update if needed where it is being used currently.